cram/cram_codecs.c

```diff
@@ -558,7 +558,7 @@ cram_codec *cram_beta_decode_init(char *data, int size,
     else if (option == E_BYTE_ARRAY || option == E_BYTE)
 	c->decode = cram_beta_decode_char;
     else {
-	fprintf(stderr, "BYTE_ARRAYs not supported by this codec\n");
+	hts_log_error("BYTE_ARRAYs not supported by this codec");
```

Comment:
See also the last paragraph of e770a1b390bc72ba467831a67ffb2d4e1c2091f4, though the need for aligning _cram/*_ formatting with that of the same code in io_lib may have since changed.